stand/common/modinfo.h

```diff
@@ -6,6 +6,11 @@
 #ifndef COMMON_MODINFO_H
 #define COMMON_MODINFO_H
 
+extern const char *md_modtype;
```

Comment:
Same question, why not use the array type directly, avoiding pointer dereference?